orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java

```diff
@@ -543,6 +545,11 @@ private void initializePublicationDate(WorkForm w) {
         validateWork(workForm);
         if (workForm.getErrors().size() == 0) {
             removeEmptyExternalIdentifiers(workForm);
+            if (isRecordHolderInContributors(workForm)) {
```

Comment:
@DanielPalafox here you are missing a `!` right? it should be "if record holder is NOT in contributors"

Reply:
done! I rename the method